control-plane/grpc/src/operations/volume/mod.rs

```diff
@@ -0,0 +1,9 @@
+#![warn(missing_docs)]
```

Comment:
:+1: good idea

Reply:
could we add this to the lib.rs instead? then it applies to whole package?

Reply:
Yeah. I wanted to do that in a separate PR as that would need addition of comments in other files as well, not related to volume ops.